src/mllp-client/mllp_client.go

```diff
@@ -3,32 +3,46 @@ package main
 import (
 	"flag"
 	"fmt"
+	"io/ioutil"
+	"log"
 	"net"
 	"os"
-	"io/ioutil"
 )
 
 const (
-	empty_filename = "<filename>"
-	mllp_start = 0x0b
-	mllp_end1  = 0x1c
-	mllp_end2  = 0x0d
+	emptyFilename  = "<filename>"
+	emptyDirectory = "<empty_dir>"
+	mllpStart      = 0x0b
+	mllpEnd        = 0x1c
+	mllpEnd2       = 0x0d
 )
 
 func main() {
 	host := flag.String("host", "localhost", "hostname of MLLP server, default value is localhost")
 	port := flag.Int("port", 2575, "portnumber of MLLP server, default value is 2575")
-	file := flag.String("file", empty_filename, "path to file which contents will be send to the MLLP server")
+	file := flag.String("file", emptyFilename, "path to file which contents will be send to the MLLP server")
+	dir := flag.String("dir", emptyDirectory, "path to file which contents will be send to the MLLP server")
```

Comment:
`path to file` should be `path to dir`

Reply:
Thank you, I overlooked this one. I've made the fix.